tasks/release.js

```diff
@@ -1,88 +1,63 @@
 module.exports = function (grunt) {
-  var readline = require('readline');
-  var url = require('url');
-  var fs = require('fs');
-  var _ = require('lodash');
+  const readline = require('readline');
+  const url = require('url');
+  const fs = require('fs');
+  const path = require('path');
+  const _ = require('lodash');
 
   // build, then zip and upload to s3
   grunt.registerTask('release', [
     '_release:confirmUpload',
-    '_release:loadS3Config',
     'build',
-    '_release:setS3Uploads',
-    's3:release',
+    '_release:loadS3Config',
+    'aws_s3:staging',
     '_release:complete'
   ]);
 
   grunt.registerTask('_release:confirmUpload', function () {
-    var rl = readline.createInterface({
+    const rl = readline.createInterface({
       input: process.stdin,
       output: process.stdout
     });
 
     rl.on('close', this.async());
     rl.question('Do you want to actually upload the files to s3 after building?, [N/y] ', function (resp) {
-      var debug = resp.toLowerCase().trim()[0] !== 'y';
-      grunt.config.set('s3.release.debug', debug);
+      const debug = resp.toLowerCase().trim()[0] !== 'y';
+
+      grunt.config.set('aws_s3.staging.options.debug', debug);
+
       rl.close();
     });
   });
 
   // collect the key and secret from the .aws-config.json file, finish configuring the s3 task
   grunt.registerTask('_release:loadS3Config', function () {
-    var config = grunt.file.readJSON('.aws-config.json');
-    grunt.config('s3.options', {
-      key: config.key,
-      secret: config.secret
+    const config = grunt.file.readJSON('.aws-config.json');
+
+    grunt.config('aws_s3.options', {
+      accessKeyId: config.key,
+      secretAccessKey: config.secret,
+      bucket: config.bucket || grunt.config.get('aws_s3.config.bucket'),
```

Comment:
Can we store `.aws-config.json` outside of the kibana directory (ie. home directory)?  That's how the official aws sdk works, and it is awesome.  Then we can have everyone add a `kibana-staging` bucket to their home directory and lean on that by default.

To be clear: I don't think we need to or even should make that change in this PR, I'm just keen in general on getting rid of semi-permanent dev artifacts from the working directory.